pkg/autodiscovery/autoconfig.go

```diff
@@ -254,8 +252,13 @@ func (ac *AutoConfig) schedule(configs []integration.Config) {
 	ac.scheduler.Schedule(configs)
 }
 
-// storeAndResolve store (in template cache) and resolves a given config into a slice of resolved configs
-func (ac *AutoConfig) storeAndResolve(config integration.Config) []integration.Config {
+// unschedule takes a slice of configs and unschedule them
+func (ac *AutoConfig) unschedule(configs []integration.Config) {
```

Comment:
If you add this can you also call it in `processRemovedConfigs`?

Reply:
👍